app/constants/index.ts

```diff
@@ -6,10 +6,20 @@ export const ENV = (process.env.NODE_ENV ?? 'production') as Environments;
 
 export const BUY_STX_URL = 'https://coinmarketcap.com/currencies/blockstack/markets';
 
+export const STATUS_PAGE_URL = 'http://status.test-blockstack.com';
+
+export const DEFAULT_STACKS_NODE_URL = 'https://stacks-node-api.krypton.blockstack.org';
+
 export const NETWORK = process.env.STX_NETWORK;
 
-export const STATUS_PAGE_URL = 'http://status.test-blockstack.com';
+export const MAX_STACKING_CYCLES = 12;
```

Comment:
Is this just arbitrary?

Reply:
Nope, I believe it's 12 cycles max https://github.com/blockstack/stacks-blockchain/blob/master/sip/sip-007-stacking-consensus.md#submitting-reward-address-and-chain-tip-signaling

> These signed messages are valid for at most 12000 Bitcoin blocks (12 reward cycles, or ~3 month)